### PR TITLE
feat: set up ngrx-store

### PR DESCRIPTION
**Pull request by Vsevolod1020**

### CODE REVIEW

File: src/app/store/effects/app.effects.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { Injectable } from '@angular/core';
+import { Actions, createEffect } from '@ngrx/effects';
+
+
+
+@Injectable()
+export class AppEffects {
+
+
```
Comment (glebek2h) [inline src/app/store/effects/app.effects.ts:9]: please run prettier before commit or setup git hooks to do it automatically

File: src/app/store/actions/app.actions.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import { createAction, props } from '@ngrx/store';
+
+export const TestApp = createAction(
+  '[App] Test App'
```
Comment (glebek2h) [inline src/app/store/actions/app.actions.ts:4]: move string to constant

**Review by glebek2h — APPROVED** [review]

File: src/app/store/effects/app.effects.ts (hunk 1 above)
Comment (Vsevolod1020) [reply src/app/store/effects/app.effects.ts:9]: Ok

File: src/app/store/actions/app.actions.ts (hunk 2 above)
Comment (Vsevolod1020) [reply src/app/store/actions/app.actions.ts:4]: In the future I will, but it is just a test action.

**Pull request merged**